src/csharp/Microsoft.Spark.E2ETest/Resources/people.json

```diff
@@ -1,3 +1,3 @@
-{"name":"Michael"}
-{"name":"Andy", "age":30}
-{"name":"Justin", "age":19}
+{"name":"Michael", "ages":[19], "info":{"city":"Burdwan", "state":"Paschimbanga"}}
```

Comment:
Can we use some meaningful data here? What does it mean by having multiple ages?

Reply:
Make sense. I have updated accordingly. Please let me know if you still see an issue. Thanks!